### PR TITLE
Add hungarian translation

### PR DESCRIPTION
**Pull request by milli05**
Hi!
I have just added hungarian translation

### CODE REVIEW

**Comment by hivokas** [comment]
`.idea` folder should not be committed

**Comment by milli05** [comment]
Yes, sorry, I inckuded that by accident, shall I do another request or, can you use it like this?

**Comment by hivokas** [comment]
It would be better to remove this folder.
You can update this pull request.

**Comment by milli05** [comment]
I have deleted the folder.

File: resources/lang/hu/validation.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    | Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines contain the default error messages used by
+    | the validator class. Some of these rules have multiple versions such
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+
+    'attached' => 'This :attribute is already attached.',
```
Comment (hivokas) [inline resources/lang/hu/validation.php]: Translate these lines too, please.

**Comment by milli05** [comment]
I have translated those two lines as well.

**Comment by hivokas** [comment]
Thanks a lot for the contribution!

**Pull request merged**